activerecord/test/models/review.rb

```diff
@@ -0,0 +1,3 @@
+class Review < ActiveRecord::Base
```

Comment:
Eventually I found such a polymorphic association and use it. Thanks. My PR got better due to your suggestions. @matthewd